records_mover/utils/json_schema.py

```diff
@@ -52,7 +52,8 @@ def is_optional_type(tp: PythonType) -> bool:
 def is_impractical_type(python_type: PythonType) -> bool:
     # can't write code and pass it around in JSON!
     return (is_callable_type(python_type) or
-            type(python_type) == enum.EnumMeta)
+            type(python_type) == enum.EnumMeta
+            or type(python_type) == typing.ForwardRef)
```

Comment:
dont understand this change

Reply:
Shoot, I should have documented it at the time because now I don't remember.

Reply:
Okay, this is the error I get without the ForwardRef:
```
NotImplementedError: Teach me how to handle Python type ForwardRef('Connection') (parameter: db_conn)
```

Reply:
So I do want to be able to pass connection strings around as far as I understand it, I'm not sure if this is new behavior or why I had to implement this now. I can delve deeper but if you're alright, I've just added an explainer as to why now